homeassistant/components/automation/logbook.py

```diff
@@ -12,9 +12,12 @@ def async_describe_events(hass, async_describe_event):  # type: ignore
     @callback
     def async_describe_logbook_event(event):  # type: ignore
         """Describe a logbook event."""
+        message = "has been triggered"
+        if ATTR_SOURCE in event.data:
+            message = f"{message} by {event.data[ATTR_SOURCE]}"
         return {
             "name": event.data.get(ATTR_NAME),
-            "message": "has been triggered",
+            "message": message,
```

Comment:
We currently only consume 'name' in logbook.  We can likely start consuming 'message' in the future.